packages/atlas/src/joystream-lib/extrinsics.ts

```diff
@@ -465,7 +469,105 @@ export class JoystreamLibExtrinsics {
       block,
     }
   }
+  /*
+    === Channel payouts ===
+  */
+
+  getClaimableReward = async (
+    channelId: string,
+    cumulativeRewardEarned: StringifiedNumber | null,
+    nodeEndpoint: string,
+    payloadDataObjectId: string
+  ) => {
+    const payoutProof = await channelPayoutProof(
+      'URL',
+      `${nodeEndpoint}/api/v1/files/${payloadDataObjectId}`,
+      Number(channelId)
+    )
+
+    const cashout = new BN(payoutProof.cumulativeRewardEarned).sub(new BN(cumulativeRewardEarned || 0))
+    return cashout.toString()
+  }
```

Comment:
This isn't an extrinsic actually, maybe we should move it to `utils` or something?